src/components/Text/README.md

```diff
@@ -31,18 +31,18 @@ export default {
-| -------------- | -------- | ----------- | ------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
-| element        | `string` | `'p'`       | `p`, `span`, `div`, `li`                                      | HTML Element wrapper                                                                        |
-| size           | `number` | —           | `1`, `0`, `-1`, `-2`                                          | Size of text                                                                                |
-| font-family    | `string` | —           | —                                                             | Font family                                                                                 |
-| font-weight    | `number` | —           | `100`, `200`, `300`, `400`, `500`, `600`, `700`, `800`, `900` | Font weight with standard numeric keyword values                                            |
-| font-size      | `string` | —           | —                                                             | Font size, as a valid CSS value. This overrides the 'size' prop, and disables type scaling. |
-| line-height    | `number` | —           | —                                                             | Line Height, as a valid CSS value. This overrides the internally calculated line-height.    |
-| color          | `string` | —           | —                                                             | Color                                                                                       |
-| font-style     | `string` | `'inherit'` | `inherit`, `normal`, `italic`                                 | font style                                                                                  |
-| text-transform | `string` | `'inherit'` | `inherit`, `none`, `uppercase`                                | text transform                                                                              |
-| text-align     | `string` | `'inherit'` | `inherit`, `left`, `right`, `center`                          | text align                                                                                  |
+| Prop           | Type     | Default            | Possible values                                               | Description                                                                                 |
+| -------------- | -------- | ------------------ | ------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| element        | `string` | `DEFAULT_TAG_NAME` | `p`, `span`, `div`, `li`                                      | HTML Element wrapper                                                                        |
```

Comment:
Not sure if this is preferable, or there is a way to get the auto-documenter to not do this. I extracted `p` into a const since we're using it in two places now and it should be consistent, but I can revert that if it's not worth this headache

Reply:
idk how to make the auto-documenter to handle this correctly, let's just stick to using `'p'` literals instead of a const variable